### ISSUE TITLE
Accessibility of CheckboxItem when using a screen reader: Unable to describe state or action

### ISSUE THREAD

**Issue by qbalsdon** [JetBrains/compose-multiplatform#2677]
The `CheckboxItem` defined in Menu.desktop.kt gives no indication:
1. That it is a checkbox, i.e. toggle-able
2. When unchecked, that it is not selected. However it does announce the tick on highlight, but that requires the user to come back and confirm

I think the Role, Name and Value should be clearer (Menu Item, Checkbox, Unchecked, [Name], [Action Description])

This does not just apply to `CheckboxItem` however, all menu items could use this.

I think for JC for Desktop the Menu is specifically critical for the accessibility of the UX, because even if the UI is not as accessible as it could be, the menu items and keyboard shortcuts could be the "easy win"

**Comment by dima-avdeev-jb** [comment]
Thanks for the issue!

**Comment by kirill-grouchnikov** [comment]
Is there a `MenuItem` role? It's not at https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/semantics/SemanticsProperties.kt;l=583

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2677">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
